spec/requests/recruitment_cycles_spec.rb

```diff
@@ -1,30 +1,42 @@
 require 'rails_helper'
 
 describe 'Recruitment cycles' do
+  let(:provider) { jsonapi(:provider) }
+
   before do
     stub_omniauth
+    stub_api_v2_request("/providers/#{provider.provider_code}", provider.render)
     get(auth_dfe_callback_path)
   end
 
   describe 'GET show' do
     it 'redirects to the course index page' do
-      provider = jsonapi(:provider)
-      stub_api_v2_request("/providers/#{provider.provider_code}", provider.render)
       allow(Settings).to receive(:rollover).and_return(false)
 
       get("/organisations/#{provider.provider_code}/2019")
-      expect(response).to redirect_to(provider_recruitment_cycle_courses_path(provider.provider_code, '2019'))
+      expect(response).to redirect_to(provider_path(provider.provider_code))
+
+      get("/organisations/#{provider.provider_code}/2020")
+      expect(response).to redirect_to(provider_path(provider.provider_code))
     end
 
     context 'rollover' do
       it 'renders the recruitment cycle page' do
-        provider = jsonapi(:provider)
-        stub_api_v2_request("/providers/#{provider.provider_code}", provider.render)
         allow(Settings).to receive(:rollover).and_return(true)
 
         get("/organisations/#{provider.provider_code}/2019")
         expect(response.body).to include('Current cycle (2019 – 2020)')
       end
     end
   end
+
+  describe "when visiting a cycle year that doesn’t exist" do
+    scenario 'it 404s' do
+      get("/organisations/#{provider.provider_code}/1999")
+      expect(response.code).to eq("404")
```

Comment:
Would the following work in this context?

```suggestion
      expect(response).to have_http_status(:not_found)
```